source/docs/advanced/payments/payline.md

```diff
@@ -0,0 +1,114 @@
+---
+id: payline
+title: Monext Online integration
+---
+
+This page contains information about the different ways you can accept payments with [Monext Online (Payline)](https://www.monext.fr/online) in your Front-Commerce application.
+
+<blockquote class="important">
+  **EARLY PREVIEW:** _This payment method is currently in early preview. It will be production ready in Front-Commerce `2.6.0`_
+</blockquote>
+
+
+## Magento1 module
+
+<blockquote class="feature--new">
+  _This feature has been added in versions: Front-Commerce `2.5.0`_
+</blockquote>
+
+<blockquote class="note">
+**Note** This integration is aimed at being transparent for administrators and developers. That is why we haven't duplicated documentation from existing Magento resources. Please [contact us](mailto:contact@front-commerce.com) if you need further assistance.
+</blockquote>
+
+Front-Commerce for Magento 1 requires a `FrontCommerce_Payline` module that turns the [Monext Payline official Magento 1 extension](https://docs.monext.fr/display/DT/Plugin+Magento+1) into a headless payment method for Front-Commerce. It is aimed at being transparent for Magento administrators and developers while allowing for a better Customer experience in a Front-Commerce application.
```

Comment:
I'd add a link to the FrontCommerce_Payline module here (even if the installation is described below).